Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /package
 
 COPY . ./
 
+RUN rm -rf /package/.build/debug
```

Comment:
this seems to have fixed the symlink error we get when run tests on linux+docker.